core/unit_test/TestViewAPI.hpp

```diff
@@ -1571,51 +1571,6 @@ class TestViewAPI {
     typename multivector_type::const_type cmvX(cmv);
     typename const_multivector_type::const_type ccmvX(cmv);
   }
-
-  static void run_test_error() {
-// FIXME_MSVC_WITH_CUDA
-// This test doesn't behave as expected on Windows with CUDA
-#if defined(_WIN32) && defined(KOKKOS_ENABLE_CUDA)
-    if (std::is_same<typename dView1::memory_space,
-                     Kokkos::CudaUVMSpace>::value)
-      return;
-#endif
```

Comment:
Noting that this was never true in the current develop but I explicitly instantiated the new test on shared memory spaces so we will see if that is still an issue.